docs/minishift-installation/master.adoc

```diff
@@ -18,9 +18,9 @@ include::topics/proc_starting-and-configuring-the-openshiftlocal-for-the-launche
 include::topics/proc_creating-a-github-personal-access-token.adoc[leveloffset=+1]
 
 [[create-launcher-app]]
-== Creating the {launcher} Service
+== Creating the {launcher} Tool
 
-This chapter describes multiple ways of creating the {launcher} service.
+This chapter describes multiple ways of creating the {launcher} tool.
```

Comment:
This isn't really "creating", is it? I guess we're intentionally careful about wording here, but I think this doesn't describe how to "create" the tool, but how to _deploy_ the dool.

Reply:
Or _install_. I've seen in this PR that we use both.